openapi.yaml

```diff
@@ -4670,22 +4676,21 @@ components:
           items:
 # {% if False %}
             allOf:
-# {% else %}{{ "\n" }}            oneOf:{% endif %}
+# {% else %}{{ "\n" }}            anyOf:{% endif %}
```

Comment:
We need oneOf here because the value cannot be an integer and a string at the same time.